tests/all-versions.test.js

```diff
@@ -23,59 +23,58 @@ const testVersions = [
   ['ember-cli-latest'],
 ];
 
-describe('Test on all Ember versions', function () {
-  for (let [version, packages] of testVersions) {
-    it(`should work for ember version ${version}`, async function ({ expect }) {
-      let tmpobj = tmp.dirSync({ unsafeCleanup: true });
-
-      const path = dirname(require.resolve(version));
-      const cliPath = join(
-        dirname(await packageUp({ cwd: path })),
-        'bin',
-        'ember',
-      );
-
-      await execaNode({
-        cwd: tmpobj.name,
-      })`${cliPath} new test-app --skip-npm --pnpm`;
-
-      const cwd = join(tmpobj.name, 'test-app');
+async function generateEmberApp(tmpDir, version, packages) {
+  console.log(`🤖 generating ember app for version ${version} 🐹`);
 
-      await execa({ cwd })`pnpm i`;
+  const path = dirname(require.resolve(version));
+  const cliPath = join(dirname(await packageUp({ cwd: path })), 'bin', 'ember');
+  const cwd = join(tmpDir, 'test-app');
 
-      if (packages?.length) {
-        await execa({ cwd })`pnpm i ${packages.join(' ')}`;
-      }
+  await execaNode({
+    cwd: tmpDir,
+  })`${cliPath} new test-app --skip-npm --pnpm`;
+  await execa({ cwd })`pnpm i`;
+  if (packages?.length) {
+    await execa({ cwd })`pnpm i ${packages.join(' ')}`;
+  }
 
-      // write out a basic acceptance test
-      await mkdir(join(cwd, 'tests/acceptance'));
-      await writeFile(
-        join(cwd, 'tests/acceptance/index-test.js'),
-        await readFile('./tests/fixtures/index-test.js', 'utf-8'),
-      );
+  const fixture = fixturify.readSync('./tests/fixtures');
+  fixturify.writeSync(cwd, fixture);
+}
 
-      await execa({ cwd, stdio: 'inherit' })`${cliPath} build`;
-      let { stdout } = await execa({ cwd })`${cliPath} test --path dist`;
-      console.log(stdout);
+async function testEmber(cwd, expect) {
+  console.log('🤖 testing ember app 🐹');
+  await execa({ cwd, stdio: 'inherit' })`npm run build`;
+  let { stdout } = await execa({ cwd })`npm run test:ember`;
+  console.log(stdout);
 
-      expect(stdout).to.include('# fail  0');
+  expect(stdout).to.include('# fail  0');
+}
 
-      // this prevents the script from erroring because of this
-      await execa({ cwd })`pnpm uninstall ember-fetch`;
+async function runCodemod(cwd) {
+  console.log('🤖 running ember-vite-codemod 🐹');
+  // this prevents the script from erroring because of this
```

Comment:
This suggestion comes a bit late; I've been thinking about this comment since we merged it.

```suggestion
  // ember-fetch is part of the classic app blueprint, but 
  // removing it is a prerequisite to running the codemod.
```